Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -414,20 +408,53 @@ public void finish() {
     @Override
     protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
+        switch (requestCode) {
```

Comment:
Could we have a single `DropInClient#handleActivityResult()` that does this switch internally and forwards to the appropriate client based on request code?

Reply:
Good call - I moved it all into one `handleActivityResult` method that calls the `DropInClient` methods.